fractal-puppy/Engine
# New skill: Machete Dance

This PR contains:
- A new skill machete dance! Spawn some deadly spinning machetes around the player.
- Cooldown to every skill.
- New cheat: Press F9 to disable skills cooldown.

[Example](https://gyazo.com/1da931b11ea200d3f0640c9a2f6ae245)

**To test:**
- Load the first level.
- Level up.
- Unlock the new skill (in the passive column).
- Try it against enemies, see if it looks something similar to the example and if it damages the enemies.
- Unlock more skills and see if they have a cooldown.
- Press F8 and see if now there is no cooldown after a skill is used.
